### PR TITLE
Add scripts to help running test services

### PR DESCRIPTION
**Pull request by adericbourg**
Includes:
* Cassandra
* Carbon
* Graphite Web

### CODE REVIEW

File: scripts/cassandra_env.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cd $(git rev-parse --show-toplevel)
```
Comment (iksaif) [inline scripts/cassandra_env.sh]: add a shebang ?

And a top-file comment to explain what it does and how to use it. Also applies to other file.

File: scripts/run_graphite_web.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+cd $(git rev-parse --show-toplevel)
+source scripts/env.sh
+
+GRAPHITE_WEB_CONF="${BG_VENV}/lib/python2.7/site-packages/graphite/local_settings.py"
```
Comment (iksaif) [inline scripts/run_graphite_web.sh]: `python2.7` depends on the default python binary, which might be python 3

File: scripts/cassandra_env.sh (hunk 1 above)
Comment (adericbourg) [reply scripts/cassandra_env.sh]: Done.

File: scripts/run_graphite_web.sh (hunk 2 above)
Comment (adericbourg) [reply scripts/run_graphite_web.sh]: Done.
(but unsure that I did it in a correct way)

**Pull request merged**